int-set/src/lib.rs

```diff
@@ -246,6 +272,34 @@ impl<T: Domain<T>> IntSet<T> {
         }
     }
 
+    /// Returns true if this set contains at least one element in 'range'.
+    pub fn intersects(&mut self, range: RangeInclusive<T>) -> bool {
```

Comment:
I would expect an `intersects` method to take another set, and would call this method `intersects_range` 🤷